ctapipe/image/extractor.py

```diff
@@ -532,13 +550,17 @@ def _calculate_correction(self, telid):
             self.window_shift.tel[telid],
         )
 
-    def __call__(self, waveforms, telid, selected_gain_channel):
+    def __call__(
+        self, waveforms, telid, selected_gain_channel, broken_pixels
```

Comment:
Do you want to change these to `tel_id` ?

Reply:
I'll make a follow up PR updating everything at once